### ISSUE TITLE
Capital One Mobile

### ISSUE THREAD

**Issue by DetColumbo** [PrivSec-dev/banking-apps-compat-report#107]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### App name

Capital One Mobile

### Link to app

https://play.google.com/store/apps/details?id=com.konylabs.capitalone

### App version

v5.98.7

### Country of the app

United States

### Build Number

SQ3A.220705.003.A1.2022071300

### Device

- [X] Pixel 6 Pro
- [ ] Pixel 6
- [ ] Pixel 5a
- [ ] Pixel 5
- [ ] Pixel 4a (5G)
- [ ] Pixel 4a
- [ ] Pixel 4 XL
- [ ] Pixel 4
- [ ] Pixel 3a XL
- [ ] Pixel 3a
- [ ] Pixel 3 XL
- [ ] Pixel 3

### Google Play installed

- [X] Installed
- [ ] Not installed

### Google Play services Network permission revoked?

- [ ] Revoked
- [X] Not revoked
- [ ] I did not have Google Play services installed

### SafetyNet Enforcement

- [ ] Enforced
- [ ] Not enforced
- [X] Unsure

### Native code debugging

- [ ] Enabled
- [ ] Disabled

### Stock OS compatibility

- [ ] Works
- [ ] Does not work
- [X] Not tested

### Profile app tested in

- [X] Main user profile
- [ ] Secondary user profile
- [ ] I used a work profile device manager app (Please mention more details about it down below like the app)

### Are there any extra notes you think users should know about?

`Exploit protection compatibility mode` needs to be enabled.

**Comment by git-diablo2424** [comment]
I am having this same issue with my Pixel 6 running GrapheneOS vSQ3A.220705.004.2022080900 and Capital One v5.100.11. Attached is a logcat of the app crashing, I have given Capital One app all permissions, but it still crashes. I do have GrapheneOS sandboxed Google Play Services installed, and installed Capital One from the Google Play Store.

[CapitalOne_Crash_Logcat3.txt](https://github.com/akc3n/banking/files/9337765/CapitalOne_Crash_Logcat3.txt)

**Comment by akc3n** [comment]
> I am having this same issue with my Pixel 6 running GrapheneOS vSQ3A.220705.004.2022080900 and Capital One v5.100.11. Attached is a logcat of the app crashing, I have given Capital One app all permissions, but it still crashes. I do have GrapheneOS sandboxed Google Play Services installed, and installed Capital One from the Google Play Store.
> 
> [CapitalOne_Crash_Logcat3.txt](https://github.com/akc3n/banking/files/9337765/CapitalOne_Crash_Logcat3.txt)

@git-diablo2424 
Okay, thank you. I will take a look and get back to you.

**Comment by akc3n** [comment]
> Crashes 3 seconds after opening. Requests permission to access calls upon installation. Once permission is granted it crashes and then will only crash when reopened. Prior version (i.e. before Capital One updated the app on July 5) worked perfectly.

@DetColumbo 
Thanks for the info. I'll have to investigate and get back to you on this matter.

**Comment by akc3n** [comment]
Have you tried enabling `Native Code debugging` from your owner profile in the security settings menu @DetColumbo @git-diablo2424

**Comment by git-diablo2424** [comment]
I'm new to debugging, but I think I did it right, this should be the logcat with Native Code debugging enabled? If not, I can try again.
[CapitalOne_Crash_Logcat4.txt](https://github.com/akc3n/banking/files/9418381/CapitalOne_Crash_Logcat4.txt)

**Comment by akc3n** [comment]
@git-diablo2424 and @DetColumbo 
I hope that sooner than later, with this repo being moved, your log cat reports may get reviewed.

**Comment by Asara** [comment]
I think this issue can be closed.
I was finally able to log in after months after enabling `Exploit protection compatibility mode`

Under Capital One -> App Info -> Advanced -> Exploit protection compatibility mode.  No more crashes on startup!

**Comment by tommytran732** [comment]
Confirmed working, I will edit the report and add this to the list.

**Comment by derhornspieler** [comment]
Capital One app crashes with GrapheneOS. Should be crossed out as not usable.

**Comment by lake-hope** [comment]
yeah, needed to disable exploit protection compatibility. ironic.
